static/js/app.js

```diff
@@ -276,10 +276,54 @@ $(function() {
             return params;
         },
         setMultipleMarkersIcon: function() {
+            var markerGroups = [];
+            var markerObj;
+            var groupID = 1;
+            var groupsColours = [];
             _.each(this.oms.markersNearAnyOtherMarker(), function(marker) {
                 marker.icon = getIcon("multiple", SEVERITY_VARIOUS);
                 marker.title = 'מספר תאונות בנקודה זו';
+
+                markerObj = {
+                    marker: marker,
+                    group: false
+                };
+                markerGroups.push(markerObj);
             });
+            _.each(markerGroups, function(m){
+                if (!m.group){
+                    m.group = groupID;
+                    groupsColours.push(m.marker.view.model.get('severity'));
+                    _.each(this.oms.markersNearMarker(m.marker), function(markerNear){
+                        for (var i = 0;i < markerGroups.length;i++){
+                            if (markerGroups[i].marker.id == markerNear.id){
+                                markerGroups[i].group = groupID;
+                                break;
+                            }
+                        }
+                        if ( (groupsColours[groupsColours.length -1] != markerNear.view.model.get('severity')) ){
+                            groupsColours[groupsColours.length -1] = 0;
+                        }
+                    });
+                    groupID++;
+                }
+            }.bind(this));
+            _.each(markerGroups, function(m){
+                switch(groupsColours[m.group -1]){
+                    case 0:
+                        m.marker.icon = getIcon("multiple", SEVERITY_VARIOUS);
+                        break;
+                    case 3:
```

Comment:
After looking at:
```javascript
var MULTIPLE_ICONS = {};
MULTIPLE_ICONS[SEVERITY_FATAL] = "/static/img/icons/multiple_lethal.png";
MULTIPLE_ICONS[SEVERITY_SEVERE] = "/static/img/icons/multiple_severe.png";
MULTIPLE_ICONS[SEVERITY_LIGHT] = "/static/img/icons/multiple_medium.png";
MULTIPLE_ICONS[SEVERITY_VARIOUS] = "/static/img/icons/multiple_various.png";
```
and
```javascript
function getIcon(accidentType, severity) {
...
}
```

You're right. I've deleted the 'switch' statement and replaced it with a one line icon assignment.